students/jjakub/session03/slicing_lab.py

```diff
@@ -0,0 +1,38 @@
+
+a_vec = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15]
+a_str = "this is a string"
+
+def first_last(seq):
+    return seq[-1:]+ seq[1:-1]+ seq[:1]
+
+def everyother(seq):
+    return seq[0::2]
+
+def mid_everyother(seq):
+    return seq[4:-4:2]
+
+def reverse(seq):
+    return seq[::-1]
+
+def thirds(seq):
+    thrd = len(seq)//3
+    return seq[-thrd:] + seq[0:thrd + thrd]
```

Comment:
I need to refer you to my same comment on another PR for this code: [here](https://github.com/UWPCE-PythonCert-ClassRepos/Au2018-Py210B/pull/42#discussion_r228295093). It looks like you two are somehow sharing incorrect code with each other, and that's not very good. Please do work on your code by yourself. This will have to be strictly enforced from the next assignment on.

/cc @Saravji and @aminavi FYI as well.

Reply:
Thanks for the feedback. We were collaborating on the homework during the in class work sessions. I didn't realize it would be an issue. Apologies.

Reply:
Collaboration is fine and not an issue, but sharing code is discouraged, and especially sharing incorrect code may raise concerns. When collaborating, please make sure to understand what collaborated code means and how it works.